### PR TITLE
makes epipens more powerful at stopping losebreath

### PR DESCRIPTION
**Pull request by Iamgoofball**
fixes unreported issue of "epipens are fucking worthless"

basically it now does literally everything inaprov did

### CODE REVIEW

**Comment by Deantwo** [comment]
Don't see how ```M.losebreath = max(10, M.losebreath-5)``` is more powerful then plain ```M.losebreath = 3```, but sure?

File: code/modules/reagents/Chemistry-Goon-Medicine.dm [hunk 1]
```diff
@@ -567,8 +567,8 @@ datum/reagent/epinephrine/on_mob_life(var/mob/living/M as mob)
 		M.adjustFireLoss(-1*REM)
 	if(M.oxyloss > 35)
 		M.setOxyLoss(35)
-	if(M.losebreath > 3)
-		M.losebreath = 3
+	if(M.losebreath >= 10)
```
Comment (Deantwo) [inline code/modules/reagents/Chemistry-Goon-Medicine.dm]: Wouldn't ```>``` make more sense?
I mean if it is already 10, then why change it to 10?

But I guess it's no huge deal.

**Comment by paprka** [comment]
goonchem is too hardcore for me. As long as this allows people to be defibbed and stop the breathloss immediately so you can actually heal people before they die again.

**Comment by Iamgoofball** [comment]
considering this now does exactly what inaprov did and some extra, it should

**Comment by Jordie0608** [comment]
Changelog.

**Pull request merged**